beacon-node-oapi.yaml

```
@@ -161,6 +161,8 @@ paths:
     $ref: "./apis/validator/sync_committee_contribution_and_proof.yaml"
   /eth/v1/validator/prepare_beacon_proposer:
     $ref: "./apis/validator/prepare_beacon_proposer.yaml"
+  /eth/v1/validator/register_validator:
+    $ref: "./apis/validator/register_validator.yaml"
```

```suggestion
  /eth/v1/validator/register_validators:
    $ref: "./apis/validator/register_validators.yaml"
```

this does take an array so I think we should pluralize

---

Okay to resolve then to maintain consistency?

---

yeah, we can ignore this